pkg/utils/utils.go

```diff
@@ -20,6 +20,7 @@ const (
 	scriptSrc     = "script-src"
 	styleSrc      = "style-src"
 	consoleDot    = "console.redhat.com"
+	cdnSegment    = "cdn.segment.com"
```

Comment:
We should be fetching console.redhat.com hosted segment scripts and content and not loading directly from segment. We should try to figure out where this is coming from.

@jhadvig FYI

Reply:
@spadgett it looks like its coming from the telemetry plugin by default - https://github.com/openshift/console/blob/84fa3ce6fdfb328b3fe79c39be5d83a5d12dc635/frontend/packages/console-telemetry-plugin/src/listeners/segment.ts#L18
But we are using the console.redhat.com explicitly - https://github.com/openshift/console-operator/blob/main/manifests/05-telemetry-config.yaml#L14

But most probably this is cause by the kubevirt-ui plugin - https://github.com/kubevirt-ui/kubevirt-plugin/blob/0f5928ddf051bc8811cba9b8465e6db063b51a71/src/utils/extensions/telemetry/telemetry.ts#L64

Thinking if the plugin should use the same logic as we do:
```ts
const jsHost = window.SERVER_FLAGS?.telemetry?.SEGMENT_JS_HOST || 'cdn.segment.com';
```